packages/components/atoms/f-button/stories/IconButton.stories.js

```diff
@@ -1,88 +1,137 @@
-import {
-    withKnobs,
-    boolean,
-    select,
-    text
-} from '@storybook/addon-knobs';
 import { withA11y } from '@storybook/addon-a11y';
 import {
-    CrossIcon,
-    PlusIcon,
-    MoreVerticalIcon
+    CrossIcon, PlusIcon, MoreVerticalIcon
 } from '@justeat/f-vue-icons';
+import SharedButtonArgTypes from './sharedButtonArgTypes';
 import FButton from '../src/components/Button.vue';
-import { VALID_BUTTON_TYPES, VALID_BUTTON_SIZES } from '../src/constants';
 
 export default {
     title: 'Components/Atoms/f-button',
-    decorators: [withKnobs, withA11y]
+    decorators: [withA11y],
+    parameters: {
+        backgrounds: {
+            values: [
+                { name: 'White', value: '#fff' },
+                { name: 'Dark', value: '#333' },
+                { name: 'Blue', value: '#bfe6ff' }
+            ],
+            default: 'White'
+        }
```

Comment:
@xander-marjoram I've just used this to add a different background colour option so it's easier to see the icons - opted for the grey/black originally but as @dandel10n suggested the blue makes them pop more 😄 I've added the white and dark still and kept white as default - otherwise, all the stories in f-button would have the blue background